template.tpl

```diff
@@ -1265,11 +1325,22 @@ const initUserData = (evData, tagConfig) => {
  * @returns - Object
  */
 const makeGroupsProperties = (evData, tagConfig) => {
```

Comment:
This would be great! I avoided going there for this PR, but there is definitely room to improve the consistency. [Issue opened](https://github.com/snowplow/snowplow-gtm-server-side-amplitude-tag/issues/36)!